docs/getting-started/react-native-ios.md

```diff
@@ -43,8 +64,21 @@ def flipper_post_install(installer)
         cflags << '-DFB_SONARKIT_ENABLED=1'
       end
       config.build_settings['OTHER_CFLAGS'] = cflags
+      if (config.build_settings['OTHER_SWIFT_FLAGS'])
```

Comment:
This change is just relevant for Swift projects, can we have two sections in the docs, one for Objective C and one for Swift.

Reply:
Since there is a if condition, should do no harm to OC project right?

Reply:
Even OC projects have OTHER_SWIFT_FLAGS build settings.

Reply:
Even so, still sounds reasonable to use one configuration for all if no actual conflicts right? Imagine one wants to turn a stock rn project into swift, how can he know he should add one more flag without any hint? (there will be no errors at all)